Formula/openjdk.rb

```diff
@@ -89,14 +88,15 @@ def install
     args += if OS.mac?
       %W[
         --enable-dtrace
-        --with-extra-ldflags=-headerpad_max_install_names
+        --with-extra-ldflags=-headerpad_max_install_names\ -Wl,-rpath,#{loader_path}/server
         --with-sysroot=#{MacOS.sdk_path}
       ]
     else
       %W[
         --with-x=#{HOMEBREW_PREFIX}
         --with-cups=#{HOMEBREW_PREFIX}
         --with-fontconfig=#{HOMEBREW_PREFIX}
+        --with-extra-ldflags=-Wl,-rpath,#{loader_path}/server"
```

Comment:
I'd personally have a ldflags array that we `join(" ")` so we don't have this duplication.

Reply:
Yea, I considered that too. Let's do that.